Automattic/wp-calypso
# DIFM: Replaces Sitemap with Testimonials on the Page Selector Step

#### Changes proposed in this Pull Request
p1653050432966909-slack-C02KVCAL7GX
* Replaces the Sitemap page with Testimonials page on the Page Selector Step.
* Changed the colour of the thumbnails to increase contrast. p1653075691656219/1653050432.966909-slack-C02KVCAL7GX

|Before|After|
|---|---|
|<img width="858" alt="image" src="https://user-images.githubusercontent.com/5436027/169758525-6711ccec-6776-4e5a-b21c-cb6adc70bf5a.png">|<img width="877" alt="image" src="https://user-images.githubusercontent.com/5436027/169757751-0a6d710d-f91f-4eb4-b53f-dc7069a9f0e1.png">|

#### Testing instructions



* Go to `/start/do-it-for-me`
* On the Page Selector step, confirm that you can see the Testimonials page. (Please see the screenshot above)
* Select the Testimonials page and complete the product purchase.
* On the Website Content step, confirm that you can see the Testimonials page.
<img width="792" alt="image" src="https://user-images.githubusercontent.com/5436027/169758359-274366e1-a82a-4cd2-a760-2df60a537a2a.png">
